.github/workflows/linux.yaml

```diff
@@ -57,7 +57,6 @@ jobs:
       nightly: ${{ github.event.inputs.nightly }}
       modular: true
   linux-coverage:
-    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
```

Comment:
This will result in coverage running on nightlies and all branches, is that intended?

Reply:
Actually, I take it back.  It will run on nightlies, but not all branches so long as this code is in main it will run on main and feature/* branches.

Reply:
I don't think nightlies make sense. If we already run on push we have continuous tracking.

I'll update.

Reply:
Ok, reverted to the original check and added the `pull_request` event